dashboards/k8s-storage-resources-glusterfs-pv.libsonnet

```diff
@@ -0,0 +1,291 @@
+      //     g.statPanel('')
+      //   )
+      // The below variables needs to chanaged when cluster_id is introduced in the metrics
+      // local queryLabels = "{job='$OCS_Cluster', volume='$Volume'}";
+      // local fopQueryLabel = "{job='$OCS_Cluster', volume='$Volume', fop!=''}";
+      // Below variables are defined considering that there is only one gluster cluster.
+      local queryLabels = "{volume='$Volume'}";
+      local fopQueryLabel = "{volume='$Volume', fop!=''}";
+      local fopLockQueryLabel = "{volume='$Volume',fop='LOCK_OPS'}";
+      local fopInodeQueryLabel = "{volume='$Volume',fop='INODE_OPS'}";
+      local fopEntryQueryLabel = "{volume='$Volume',fop='ENTRY_OPS'}";
+      local fopReadWriteQueryLabel = "{volume='$Volume',fop='READ_WRITE_OPS'}";
+
+      // g.dashboard(
+      //   'K8s / Storage Resources / Persistent Volume / glusterfs',
+      //   uid=($._config.grafanaDashboardIDs['k8s-storage-resources-glusterfs-pv.json']),
+      // ).addTemplate('OCS_Cluster', 'gluster_volume_up', 'job')
+      // .addTemplate('Volume', 'gluster_volume_up{job = "$OCS_Cluster"}', 'volume')
+      g.dashboard(
+        'K8s / Storage Resources / Persistent Volume / glusterfs',
+        uid=($._config.grafanaDashboardIDs['k8s-storage-resources-glusterfs-pv.json']),
+      )
+      .addTemplate('Volume', 'gluster_volume_up', 'volume')
+      .addRow(
+        (g.row('Volume Summary') +
+         {
+           showTitle: false,
+           panels: self._panels,
+         })
+        .addPanel(
+          g.panel('Status') +
+          statPanelArgs +
+          g.statPanel('gluster_volume_up{volume = "$Volume"}',) +
+          {
+            valueMaps: [
+              {
+                op: '=',
+                text: 'DOWN',
+                value: '0',
+              },
+              {
+                value: '1',
+                op: '=',
+                text: 'UP',
+              },
+            ],
+            thresholds: '0.9,1.1',
+            colors: [
+              '#cc0000',
+              '#92d400',
+              '#FFFFFF',
+            ],
+            colorValue: true,
+          }
+        )
+        .addPanel(
+          g.panel('Capacity Utilization') +
+          statPanelArgs +
+          g.statPanel('gluster:volume_capacity_used_bytes_total:sum%s / gluster:volume_capacity_total_bytes:sum%s' % [queryLabels, queryLabels])
+        )
+        .addPanel(
+          g.panel('Available Capacity') +
+          statPanelArgs +
+          g.statPanel('gluster:volume_capacity_total_bytes:sum%s - gluster:volume_capacity_used_bytes_total:sum%s' % [queryLabels, queryLabels], 'decbytes')
+        )
+        .addPanel(
+          g.panel('IOPS') +
+          statPanelArgs +
+          g.statPanel('(sum (gluster_volume_profile_total_reads_interval%s) + sum(gluster_volume_profile_total_reads_interval%s))/max(gluster_volume_profile_duration_secs_interval%s)' % [queryLabels, queryLabels, queryLabels], 'none')
```

Comment:
You are adding `gluster_volume_profile_total_reads_interval` twice. I feel it should be sum of `gluster_volume_profile_total_reads_interval` and `gluster_volume_profile_total_writes_interval`

Reply:
ack